src/EnergyPlus/EconomicLifeCycleCost.hh

```diff
@@ -131,21 +194,23 @@ namespace EconomicLifeCycleCost {
         BasePeriod,
     };
 
-    enum class iSourceKind
+    enum class SourceKindType
     {
-        Unassigned,
+        Unassigned = -1,
         Recurring,
         Nonrecurring,
         Resource,
         Sum,
+        Num
     };
 
-    enum class iPrValKind
+    enum class PrValKind
     {
-        Unassigned,
+        Unassigned = -1,
         Energy,
         NonEnergy,
         NotComputed,
+        Num
     };
 
     constexpr const char *MonthNames(int const &i)
```

Comment:
This thing can be a `constexpr std::array<std::string view, 12>` lookup, it shouldn't be a function. It should also probably be in a central `utility` or `calendar` module.